ch5/problem_0.md

```diff
@@ -1 +1,4 @@
-## How would you describe method resolution order in the diamond pattern?
\ No newline at end of file
+## How would you describe method resolution order in the diamond pattern?
+If compare the Inheritance pattern to a tree structure, I would describe method resolution order in the diamond pattern as Breadth-first search.
```

Comment:
Nice, this is definitely on the right track - you could make this more clear by describing how a single method would be called if they were implemented in some or all of the classes.